MAUI/TextInputLayout/Supported-input-views.md

```diff
@@ -184,4 +184,133 @@ inputLayout.Content = new SfMaskedEntry() { MaskType = MaskedEntryMaskType.Simpl
 {% endhighlight %}
 {% endtabs %}
 
-![MaskedEntry](images/SupportedInputViews/MaskedEntry.png)
\ No newline at end of file
+![MaskedEntry](images/SupportedInputViews/MaskedEntry.png)
+
+## Picker
+
+To initialize the `Picker` control and launch it in each platform, refer to the `getting started with picker` documentation.
+
+{% tabs %}
+{% highlight XAML %}
+
+<inputLayout:SfTextInputLayout Hint="Fruit" 
+                               HelperText="Select a fruit"
+                               ContainerType="Outlined" >
+   <picker:Picker>
+        <Picker.ItemsSource>
+            <x:Array Type="{x:Type x:String}">
+                <x:String>Apple</x:String>
+                <x:String>Orange</x:String>
+                <x:String>Strawberry</x:String>
+            </x:Array>
+        </Picker.ItemsSource>
+   </picker:Picker>
+</inputLayout:SfTextInputLayout> 
+
+{% endhighlight %}
+{% highlight C# %}
+
+var inputLayout = new SfTextInputLayout();
+inputLayout.Hint = "Fruit"; 
+inputLayout.HelperText = "Select a fruit";
+inputLayout.ContainerType = ContainerType.OutLined;
+inputLayout.ContainerBackground = Colors.Transparent;
+inputLayout.Content = new SfPicker(); 
```

Comment:
Picker object initialization is different from XAML code. ItemsSource and Selection Index properties are missing

Reply:
Addressed the changes.